### ISSUE TITLE
Define aims/target audience for lesson

### ISSUE THREAD

**Issue by aragilar** [carpentries-incubator/foundational-computer-skills#8]
I mentioned this in the email chain, but I figured I should post it here also:

I can see a number of possible aims for this workshop:
 * backfill skills we expect/the workshop requires (e.g. how to unzip files, how to navigate using the gui file browser, what a web browser is)
 * provide an introduction to carpentries (e.g. using red/green stickies, one-up one-down, etherpad notes, vidconf skills given COVID)
 * help users get their environment set up (and so centralise knowledge about setup issues)
 * provide supplementary lessons on related topics (e.g. markdown as suggested in https://github.com/carpentries-incubator/foundational-computer-skills/issues/3)

I think it would be good to have a clear set of aims, with a specific target audience in mind, otherwise this could end up a bit like shell-extras, where's there's lots of standalone content but no overall structure.

**Comment by dmolesUC** [comment]
Yeah, looking at the proposed content, there's a lot of valuable stuff, but it's kind of all over the place (suggestion for starting to address that in #17).

Personally, I'm coming (late) to this mostly thinking about the first bullet point—backfilling skills that other Carpentries workshops tend to assume you already have—and that from what I've seen, newer students who grew up with smartphones, tablets, Chromebooks and cloud apps are less likely to have. I think focusing on that pre-Carpentries prep goal might help narrow it down to something that would make a good, coherent lesson.
